cellarium/cas/client.py

```diff
@@ -393,6 +393,8 @@ def annotate_anndata(
         include_dev_metadata: bool = False,
     ) -> t.List[t.Dict[str, t.Any]]:
         """
+        Deprecated. Use :meth:`annotate_matrix` instead.
```

Comment:
Do we want to use a library like [deprecated](https://pypi.org/project/Deprecated/) for this kind of thing?

Reply:
Looks pretty nice. Just got it Installed! Thanks